crypto/BRCryptoWalletManagerClient.c

```diff
@@ -1881,15 +1881,21 @@ cwmAnnounceGetTransactionsItemGEN (BRCryptoWalletManager cwm,
     cwm = cryptoWalletManagerTake (cwm);
 
     // Fundamentally, the `transfer` must allow for determining the `wallet`
-    BRGenericTransfer transfer = gwmRecoverTransfer (cwm->u.gen, transaction, transactionLength);
+    BRArrayOf(BRGenericTransfer) transfers = gwmRecoverTransfersFromRawTransaction (cwm->u.gen, transaction, transactionLength);
```

Comment:
Although the parameters don't indicate it, `transaction` is owned by the caller and not to be freed here.  Check.